### PR TITLE
Update instructions to add info about bower

### PR DESCRIPTION
**Pull request by trexdinosaur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
 
     ```
     npm install
+    ``` 
+    If you don't have bower installed global, run the following:
```
Comment (RishabhJain2018) [inline README.md]: Can you please change the text to -> `Install bower(1.8.0) globally by running:`
 `npm install -g bower`

File: README.md [hunk 2]
```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
...
+    ```
+    npm install -g bower
+    ```
+    Finally, to install browser dependencies, run:
```
Comment (RishabhJain2018) [inline README.md]: Can you please change the text to -> `Now, install the bower dependencies`

**Comment by trexdinosaur** [comment]
Updated. Thanks!

File: README.md [hunk 3]
```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
 
     ```
     npm install
+    ``` 
+    Install bower(1.8.0) globally by running:
+    ```
+    npm install -g bower
+    ```
+    Now install the bower dependencies, run:
```
Comment (RishabhJain2018) [inline README.md]: Please change `Now install the bower dependencies, run:` -> `Now install the bower dependencies by running:`

**Comment by trexdinosaur** [comment]
Updated

**Pull request merged**